nexus/src/app/background/init.rs

```diff
@@ -505,6 +505,7 @@ impl BackgroundTasksInitializer {
             task_impl: Box::new(
                 support_bundle_collector::SupportBundleCollector::new(
                     datastore.clone(),
+                    resolver.clone(),
```

Comment:
Maybe this isn't the best approach, given the conversation in the control plane huddle today. On the other hand, we're very unlikely to be collecting bundles fast enough to have a real impact.

Reply:
I'm fine to punt on this until we sort out the progenitor integration with qorb more holistically